compiler_opt/rl/compilation_runner_test.py

```diff
@@ -107,7 +108,8 @@ def test_policy(self, mock_compile_fn):
     runner = compilation_runner.CompilationRunner(
         moving_average_decay_rate=_MOVING_AVERAGE_DECAY_RATE)
     data = runner.collect_data(
-        file_paths=('bc', 'cmd'),
+        module_spec=corpus.ModuleSpec(
+            _exec_cmd=('-O2',), _xopts={}, name='dummy'),
```

Comment:
rename `_exec_cmd` to `exec_cmd`, because it's now part of the public interface; `_xopts` to `extra_opts` (more verbose, but more clear, too)